lib/import/tabular_basic.rb

```diff
@@ -0,0 +1,37 @@
+module Import
+  module TabularBasic
+    private 
+      # valid header fields
+      def parse_headers(row)
+        headers = []
+        $i = 0
```

Comment:
It's looks like enumerable method are more specific to a specific format like `row.cells.each` for Excel while CSV don't have that support. But row[i] supported by both formats to access a cell at location i.
How about removing the $ sign? I'll also update it use [].tap to replace the `headers` variable declaration.